### PR TITLE
Tic Tac Toe Project

### PR DESCRIPTION
**Pull request by Angery-Stoman**
Here is the documentation of my project where I will emulate the game tic tac toe.

Signed-off-by:  Teodor-Matei FRÂNCU <teodor_matei.francu@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/Index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+## Architecture 
+
+![architecture](<img width="870" alt="architecture" src="https://github.com/UPB-FILS-MA/upb-fils-ma.github.io/assets/163386844/781248c2-bc65-4d6b-a6b0-360f5891a9d4">)
+
+### Main Components
+
+Here is the list of all of the main components and a short description of their utility:
+
+-Controller
```
Comment (alexandruradovici) [inline docs/project/2024/Index.md:29]: Is this displayed correctly?

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Index.md (hunk 1 above)
Comment (Angery-Stoman) [reply docs/project/2024/Index.md:29]: how can I check if it is displayed correctly?

**Comment by Angery-Stoman** [comment]
I'm not sure if I did it correctly, as github's interface is nigh impossible for me to understand, but I think the changes should have been added.

File: docs/project/2024/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Tic-Tac_Toe
+This project aims to emulate the popular game Tic-Tac_Toe on a Rapspberry Pi.
+
+**Author**: Teodor-Matei FRÂNCU \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Angery-Stoman)
```
Comment (alexandruradovici) [inline docs/project/2024/index.md:5]:
```suggestion

:::info

**Author**: Teodor-Matei FRÂNCU \
**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Angery-Stoman)
```

:::

File: docs/project/2024/index.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+
+## Motivation
+
+I've always wanted to delve into game making, and while this may not be much, it's a small step into the world of game making. Tic Tac Toe feels simple enough for someone of my hardware knowledge level to recreate it. Choosing a game as my project also adds a level of difficulty to the journey of creating it, as I have only coded games using html, c++/c# and javascript, a.k.a. only software. This will be the first time any game that I have made has a hardware component to it.
+And as an added bonus I'm quite passionate about video games and I spend most of my free time playing them, so this project is right up my alley.
+
+## Architecture 
+
+![architecture](<img width="870" alt="architecture" src="https://github.com/UPB-FILS-MA/upb-fils-ma.github.io/assets/163386844/781248c2-bc65-4d6b-a6b0-360f5891a9d4">)
```
Comment (alexandruradovici) [inline docs/project/2024/index.md:23]: Please use the correct markdown syntax and the local file.

File: docs/project/your_curs_upb_ro_id.md
Comment (alexandruradovici) [inline docs/project/your_curs_upb_ro_id.md:1]: Please do not move this file.

File: docs/project/2024/index.md
Comment (alexandruradovici) [inline docs/project/2024/index.md:1]: Please place this file in a folder with your username.

File: docs/project/architecture.png
Comment (alexandruradovici) [inline docs/project/architecture.png:1]: Please put this file within your folder.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please delete all the file that you do not need. You your username, that is the part in front of @stud) for your folder name.

File: docs/project/2024/your_curs_upb_ro_id.md
Comment (alexandruradovici) [inline docs/project/2024/your_curs_upb_ro_id.md:1]: Do not delete this file.

File: docs/project/2024/teodor_matei.francu
Comment (alexandruradovici) [inline docs/project/2024/teodor_matei.francu:1]: This should be a folder.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/teodor_matei.francu/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+# Tic-Tac_Toe
+This project aims to emulate the popular game Tic-Tac_Toe on a Rapspberry Pi.
+
+
+:::info
+
+**Author**: Teodor-Matei FRÂNCU \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-Angery-Stoman)
+
+
```
Comment (alexandruradovici) [inline docs/project/2024/teodor_matei.francu/index.md:10]:
```suggestion
:::
```

File: docs/project/2024/teodor_matei.francu/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+-Display
+Comprised of an lcd display that will present the user the game itself, a 3x3 matrice that both the player and the computer will play on.
+
+-Buzzers
+Used to play different sounds according to the users inputs and the state of the game (loss/win/idle)
+
+### Connections
+
+-Controller to Microcontroller:
```
Comment (alexandruradovici) [inline docs/project/2024/teodor_matei.francu/index.md:46]: I don't think this renders correctly, please verify this by running `npm start`.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/teodor_matei.francu/index.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+## Architecture 
+
+![architecture](architecture.png)
+
+### Main Components
+
+Here is the list of all of the main components and a short description of their utility:
+
+-Controller
```
Comment (alexandruradovici) [inline docs/project/2024/teodor_matei.francu/index.md:32]: I don't think this renders correctly, please verify this by running `npm start`.

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/teodor_matei.francu/index.md (hunk 6 above)
Comment (Angery-Stoman) [reply docs/project/2024/teodor_matei.francu/index.md:32]: should it render as a special list of some sort?
I used "-" just for formatting, not necessarily to create a bullet point list

Comment (alexandruradovici) [reply docs/project/2024/teodor_matei.francu/index.md:32]: Did you test it?

Comment (Angery-Stoman) [reply docs/project/2024/teodor_matei.francu/index.md:32]: how exactly do I use npm start?
I searched online for a bit and I'm still unsure

Comment (alexandruradovici) [reply docs/project/2024/teodor_matei.francu/index.md:32]: Read the *Readme.md* file of the repository.

Comment (Angery-Stoman) [reply docs/project/2024/teodor_matei.francu/index.md:32]: thank you! will do

Comment (Angery-Stoman) [reply docs/project/2024/teodor_matei.francu/index.md:32]: I have tested it with Docusaurus and it looks fine on my end. 
image
This is how I intended for the site to look and it's how it got rendered.
Should it appear different?

**Review by alexandruradovici — APPROVED** [review]

**Comment by Angery-Stoman** [comment]
Is there something else that I need to do for my pull request to be accepted?

**Comment by alexandruradovici** [comment]
Just wait for another approval.

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**